src/Context/CommonContext.php

```diff
@@ -531,7 +534,9 @@ public function iCloseTheChildWindow()
     {
         $windowNames = $this->getSession()->getWindowNames();
         if (count($windowNames) > 1) {
-            $this->getSession()->stop($windowNames[1]);
+            $this->getSession()->executeScript('window.close()');
+            //switch back to main window
+            $this->getSession()->switchToWindow();
```

Comment:
- Revert changes in this method and mark is as `@deprecated`
- Add a new method with new step definition and new behaviour